Ports/sam/package.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=SAM
+version=c86ea395743b8ea4ad071c2167fd1f7f96648f7b
+files="https://github.com/vidarh/SAM/archive/${version}.tar.gz ${version}.tar.gz 1f534245e2c7a096de5f886fd96ea1ad966c4e674c1ed91e0c6a59662e8d6c11"
+auth_type=sha256
+depends="SDL2"
+
+build() {
+    export CC="${SERENITY_SOURCE_DIR}/Toolchain/Local/${SERENITY_ARCH}/bin/${SERENITY_ARCH}-pc-serenity-gcc"
```

Comment:
This should not be needed (and breaks use of the package script inside serenity). CC should already be set. Why *is* this needed?!

Reply:
I blindly copied this line from the emu2 and oksh ports. Now that I've looked deeper, setting CC in the `make` command on the next line is what allows this build to complete.